test_fms/diag_manager/test_diag_manager2.sh

```diff
@@ -891,6 +891,36 @@ printf "&diag_manager_nml \n use_modern_diag = .false. \n use_clock_average = .t
     mpirun -n 1 ../test_flexible_time
   '
 
+printf "&diag_manager_nml \n use_modern_diag = .true. \n /" | cat > input.nml
+cat <<_EOF > diag_table.yaml
+title: test_diag_manager
+base_date: 2 1 1 0 0 0
+diag_files:
+- file_name: file_openmp_test
+  freq: 1
+  freq_units: hours
+  time_units: hours
+  unlimdim: time
+  varlist:
+  - module: ocn_mod
+    var_name: var1
+    reduction: none
+    kind: r4
+  - module: ocn_mod
+    var_name: var2
+    reduction: none
+    kind: r4
+  - module: ocn_mod
+    var_name: var3
+    reduction: none
+    kind: r4
+_EOF
+
+export OMP_NUM_THREADS=2
```

Comment:
I think this should be set back to 1 or just unset entirely at the end of the script.  It would just be neater.

Reply:
https://github.com/NOAA-GFDL/FMS/pull/1281/commits/20227bad838a4b8a5c68456197a6d810212a2dba